src/libecalc/common/stream.py

```diff
@@ -15,9 +14,9 @@ class Config:
         alias_generator = to_camel_case
         allow_population_by_field_name = True
 
-    name: Optional[str]
-    rate: TimeSeriesStreamDayRate
-    pressure: TimeSeriesFloat
+    name: str
+    rate: Optional[TimeSeriesStreamDayRate]
+    pressure: Optional[TimeSeriesFloat]
     fluid_density: Optional[TimeSeriesFloat] = None
 
     def mix(self, other_stream: "Stream") -> "Stream":
```

Comment:
This method will of course become more involved/advanced when we allow self and other_stream to have different compositions.